cartridges/int_stickyio_sfra/cartridge/templates/default/checkout/customer/customerCard.isml

```diff
@@ -0,0 +1,72 @@
+<div class="card-body">
+ 	<isif condition="${pdict.allowGuestCheckout}">
+	    <form autocomplete="on" id="guest-customer" class="guest-form" method="POST" action="${URLUtils.url('CheckoutServices-SubmitCustomer')}"
+	        <isprint value=${pdict.forms.guestCustomerForm.attributes} encoding="off" /> novalidate>
+	        <fieldset class="customer-information-block">
+	            <div class="row">
+	                <div class="col-sm-12">
+	                    <div class="form-group">
+	                        <label class="form-control-label">
+	                            ${Resource.msg('field.customer.email','checkout',null)}
+	                        </label>
+	                        <input type="text" class="form-control email" id="email-guest"
+	                                    value="${(pdict.customer.registeredUser && pdict.customer.profile && pdict.customer.profile.email) ? pdict.customer.profile.email : ''}" <isprint value=${pdict.forms.guestCustomerForm.email.attributes} encoding="off"/>
+	                                    aria-describedby="guestEmailInvalidMessage" >
+	                        <div class="invalid-feedback" id="guestEmailInvalidMessage"></div>
+	                    </div>
+	                </div>
+	            </div>
+	            <isif condition="${!pdict.customer.registeredUser}">
+	                <button class="btn btn-primary btn-block submit-customer" type="submit" name="submit" value="submit-customer">
+	                    ${Resource.msg('button.guest.customer.continue', 'checkout', null)}
+	                </button>
+	            </isif>
+	             <strong>${Resource.msg('msg.guest.customer.exists', 'checkout', null)} <button type="button" class="btn btn-link js-login-customer">${Resource.msg('action.customer.login', 'checkout', null)}</button></strong>
+	        </fieldset>
+	        <input type="hidden" name="${pdict.csrf.tokenName}" value="${pdict.csrf.token}"/>
+	    </form>
+    </isif>
```

Comment:
Seems like there is an extra unneeded tab on lines 2-27.

Reply:
fixed